packages/wallet-apis/src/actions/signMessage.ts

```diff
@@ -44,12 +43,13 @@ export async function signMessage(
   signerClient: SignerClient,
   params: SignMessageParams,
 ): Promise<SignMessageResult> {
-  const account = await requestAccount(client, signerClient, {
-    accountAddress: params.account ?? client.account?.address,
-  });
-  if (!account) {
-    throw new AccountNotFoundError();
-  }
+  const accountAddress = params.account ?? client.account?.address;
+
+  const account = await requestAccount(
+    client,
+    signerClient,
+    accountAddress != null ? { accountAddress } : undefined,
```

Comment:
afaict accountAddress will either be an address or undefined right? When can it be null?